components/engine_traits/src/sst.rs

```diff
@@ -20,10 +20,10 @@ pub trait SstExt: Sized {
 }
 
 /// SstReader is used to read an SST file.
-pub trait SstReader: RefIterable + Sized {
-    fn open(path: &str) -> Result<Self>;
-    fn open_encrypted<E: EncryptionKeyManager>(path: &str, mgr: Arc<E>) -> Result<Self>;
+pub trait SstReader: RefIterable + Sized + Send {
+    fn open<E: EncryptionKeyManager>(path: &str, mgr: Option<Arc<E>>) -> Result<Self>;
```

Comment:
I'm trying to use associate type

Reply:
I will move the trait `EncryptionKeyManager` to component `encryption` to resolve the `engine_traits`  dependency of `encryption` in next PR. So we can just accept a `DataKeyManager` instead of `EncryptionKeyManager`.